tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -338,6 +338,7 @@ def test_fsdp_mixed_with_sync(world_size, tmp_path: pathlib.Path, sync_module_st
                      r'ignore:MosaicMLLogger is not in the state_dict. Its state will not be restored.:UserWarning'))
 ])
 @pytest.mark.filterwarnings(r'ignore:.*metrics are not saved with sharded state dict.*:UserWarning')
+@pytest.mark.filterwarnings(r'ignore:.*The CUDA RNG state could not be loaded.*:UserWarning')
```

Comment:
curious why need to filter this warning since this should only show up if we load and save checkpoints of different versions?

Reply:
Yep, this test does that. It saves on 2.0.1 iirc and loads on 2.1